code/drasil-printers/Language/Drasil/HTML/Print.hs

```diff
@@ -24,9 +24,9 @@ import Language.Drasil.Printing.AST (Spec, ItemType(Flat, Nested),
   ListType(Ordered, Unordered, Definitions, Desc, Simple), Expr, Fence(Curly, Paren, Abs, Norm),
   Ops(Prod, Inte, Mul, Summ, Or, Add, And, Subt, Iff, Impl, GEq, LEq, Lt, Gt, NEq, Eq,
   Dot, Cross, Neg, Exp, Not, Dim, Cot, Csc, Sec, Tan, Cos, Sin, Log, Ln, Prime, Comma, Boolean, 
-  Real, Rational, Natural, Integer, IsIn), 
+  Real, Rational, Natural, Integer, IsIn, Point), 
   Expr(Sub, Sup, Over, Sqrt, Spc, Font, MO, Fenced, Spec, Ident, Row, Mtx, Case, Div, Str, 
-  Int, Dbl), Spec(Quote, EmptyS, Ref, HARDNL, Sp, Sy, S, E, (:+:)),
+  Int, Dbl, DblSc), Spec(Quote, EmptyS, Ref, HARDNL, Sp, Sy, S, E, (:+:)),
```

Comment:
Do we need `DblSc` ? I understand that `Dbl` is printed using `showFFloat` and `DlbSc` using `showEFloat`, but the question is, can we generate `DblSc` ?

Reply:
If we don't use the build in function, we can write a similar function ourselves so that we don't need 'Dbl'. If this is what we want.